build_projects/dotnet-host-build/PkgTargets.cs

```diff
@@ -31,7 +31,9 @@ public static BuildTargetResult InitPkg(BuildTargetContext c)
             Directory.CreateDirectory(PkgsIntermediateDir);
 
             SharedHostComponentId = $"com.microsoft.dotnet.sharedhost.component.osx.x64";
-            HostFxrComponentId = $"com.microsoft.dotnet.hostfxr.component.osx.x64";
+
+            string hostFxrVersion = c.BuildContext.Get<HostVersion>("HostVersion").LockedHostFxrVersion.ToString();
```

Comment:
Have you confirmed other packages do not have this problem and that with this fix, the issue no longer repros?